docs/Tutorial/personality-switch.md

```diff
@@ -49,7 +49,7 @@ You’ll notice that apart from changing the intro of the prompt and stopwords,
 Simply put, temperature controls response creativity: the higher the temperature, the “bolder” and less deterministic the responses will be (it has a range from 0 to 1). Response length just limits the number of tokens the agent's response can have.
 
 Now it’s time to unleash your creativity! We strongly encourage you to try out different parameters and toy around with any identities for your agent that you find interesting. You can make your agent rude, nice, funny etc. 
-If you want to add a new personality, go to ```rules/nola/Conversation.drl```, and inside ```"Switch personality"``` rule, you need to do the following:
+If you want to add a new personality, you can at one or more in  ```models/models/Personality.java```, you need to do the following:
```

Comment:
What does "at one or more" mean? :) I suppose the point was "If you want to add a new personality, you can add a new entry in ```models/models/Personality.java```. Just set the following:" or something?